packages/access-api/src/service/voucher-claim.js

```diff
@@ -35,16 +35,16 @@ export function voucherClaimProvider(ctx) {
       .delegate()
 
     const encoded = delegationToString(inv)
-    // For testing
-    if (ctx.config.ENV === 'test') {
-      return encoded
-    }
 
     const url = `${ctx.url.protocol}//${ctx.url.host}/validate-email?ucan=${encoded}`
-
     await ctx.email.sendValidation({
       to: capability.nb.identity.replace('mailto:', ''),
       url,
     })
+
+    // For testing
+    if (ctx.config.ENV === 'test') {
+      return encoded
+    }
```

Comment:
Nit: Is this really necessary now hat we can provide debug `email` thing for tests ? Because ucan is already in the query param. I have changed it in other placed, must have missed it here.

Reply:
ah good question and I think no - I had originally been planning to rework the tests to get rid of this, but paused on that because we'll probably end up porting all of this over to the new session stuff, and it will make sense to rework tests and generally clean all of this up at that point.